### PR TITLE
Add new check, run flake8 to all directory, add coverage

### PR DESCRIPTION
**Pull request by mistergreen22**

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-mock
-pytest
-flake8
-git+ssh://git@github.com/threatgrid/softserve-atqc.git#egg=ctrlibrary
+mock==3.0.5
```
Comment (gdavoian) [inline test-requirements.txt:1]: Please don't fix the version of test requirements and leave everything (except the last line) as before. Our library should be compatible with Python 2.7 and 3.5-3.8, so we can't ensure that e.g. `flake8==3.8.3` exists in all the targeted versions.

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,26 @@
-dist: xenial
 language: python
+
 python:
 - '2.7'
 - '3.5'
 - '3.6'
 - '3.7'
```
Comment (gdavoian) [inline .travis.yml:7]: Please also add Python 3.8 (see https://github.com/CiscoSecurity/tr-05-api-module/blob/develop/setup.py#L64).

File: tests/unit/request/test_authorized.py [hunk 3]
```diff
@@ -50,14 +50,12 @@ def test_that_authorized_request_retrieves_token_on_expiration_and_retries():
 
     assert request.post.call_count == 2
     assert (
-        request.post.call_args_list[0] ==
-        request.post.call_args_list[1]
+            request.post.call_args_list[0] == request.post.call_args_list[1]
```
Comment (gdavoian) [inline tests/unit/request/test_authorized.py:53]: Over-indented

File: tests/functional/tests/test_ctr_05_api.py [hunk 4]
```diff
@@ -367,19 +369,21 @@ def test_python_module_positive_commands_verdict_multiple(module_tool_client):
     tool_command_hash_observable = [
         d
         for d in tool_command_response['verdicts']
-        if d['observable_value'] == SHA256_HASH and
-        d['module'] == 'Private Intelligence'
+        if d['observable_value'] == SHA256_HASH and (
+                d['module'] == 'Private Intelligence')
```
Comment (gdavoian) [inline tests/functional/tests/test_ctr_05_api.py:373]: Over-indented

File: tests/functional/tests/test_ctr_05_api.py [hunk 5]
```diff
@@ -367,19 +369,21 @@ def test_python_module_positive_commands_verdict_multiple(module_tool_client):
...
     ][0]
     tool_command_ip_observable = [
         d
         for d in tool_command_response['verdicts']
-        if d['observable_value'] == IP and
-        d['module'] == 'Private Intelligence'
+        if d['observable_value'] == IP and (
+                d['module'] == 'Private Intelligence')
```
Comment (gdavoian) [inline tests/functional/tests/test_ctr_05_api.py:379]: Over-indented

File: tests/functional/tests/test_ctia_api.py [hunk 6]
```diff
@@ -335,9 +335,9 @@ def test_python_module_ctia_positive_bundle(module_tool_client):
     assert post_tool_response['type'] == 'bundle'
     assert post_tool_response['source'] == 'ctia'
     assert post_tool_response[
```
Comment (gdavoian) [inline tests/functional/tests/test_ctia_api.py:337]:
```python
assert post_tool_response['incidents'][0]['id'] == (
    incident_post_response['id']
)
```

File: tests/functional/tests/test_ctia_api.py [hunk 7]
```diff
@@ -335,9 +335,9 @@ def test_python_module_ctia_positive_bundle(module_tool_client):
...
-        'incidents'][0]['id'] == incident_post_response['id']
+               'incidents'][0]['id'] == incident_post_response['id']
     assert post_tool_response[
-        'indicators'][0]['id'] == indicator_post_response['id']
+               'indicators'][0]['id'] == indicator_post_response['id']
```
Comment (gdavoian) [inline tests/functional/tests/test_ctia_api.py:340]: The same here

File: tests/functional/tests/test_ctia_api.py [hunk 8]
```diff
@@ -811,8 +811,8 @@ def test_python_module_ctia_positive_feed(module_headers, module_tool_client):
     assert get_tool_response['revision'] == 1
     # Get information from feed view endpoint
     assert (
-        feed.view(entity_id, get_tool_response['secret']) ==
-        {'observables': [observable]}
+        feed.view(entity_id, get_tool_response['secret']
+                  ) == {'observables': [observable]}
```
Comment (gdavoian) [inline tests/functional/tests/test_ctia_api.py:815]:
```python
assert feed.view(entity_id, get_tool_response['secret']) == (
    {'observables': [observable]}
)
```

File: test-requirements.txt (hunk 1 above)
Comment (mistergreen22) [reply test-requirements.txt]: okey

File: .travis.yml (hunk 2 above)
Comment (mistergreen22) [reply .travis.yml:7]: done

File: tests/unit/request/test_authorized.py (hunk 3 above)
Comment (mistergreen22) [reply tests/unit/request/test_authorized.py]: done

File: tests/functional/tests/test_ctr_05_api.py (hunk 4 above)
Comment (mistergreen22) [reply tests/functional/tests/test_ctr_05_api.py]: fixed

File: tests/functional/tests/test_ctr_05_api.py (hunk 5 above)
Comment (mistergreen22) [reply tests/functional/tests/test_ctr_05_api.py]: fixed

File: tests/functional/tests/test_ctia_api.py (hunk 6 above)
Comment (mistergreen22) [reply tests/functional/tests/test_ctia_api.py:340]: cool

File: test-requirements.txt [hunk 9]
```diff
@@ -1,4 +1,4 @@
 mock
```
Comment (gdavoian) [inline test-requirements.txt:1]: Please sort the requirements in the alphabetical order :)

File: tests/functional/tests/test_ctia_api.py [hunk 10]
```diff
@@ -334,8 +334,9 @@ def test_python_module_ctia_positive_bundle(module_tool_client):
     post_tool_response = bundle.export.post(payload=payload)
     assert post_tool_response['type'] == 'bundle'
     assert post_tool_response['source'] == 'ctia'
-    assert post_tool_response[
-        'incidents'][0]['id'] == incident_post_response['id']
+    assert post_tool_response['incidents'][0]['id'] == (
+        incident_post_response['id']
+    )
     assert post_tool_response[
         'indicators'][0]['id'] == indicator_post_response['id']
```
Comment (gdavoian) [inline tests/functional/tests/test_ctia_api.py:341]: Please format this assert the same way as you did with the previous one.

File: test-requirements.txt (hunk 9 above)
Comment (mistergreen22) [reply test-requirements.txt]: jawohl mein herr!

File: tests/functional/tests/test_ctia_api.py (hunk 10 above)
Comment (mistergreen22) [reply tests/functional/tests/test_ctia_api.py]: gut

**Pull request merged**